dev/vision-n-scope/7-contributions.md

```diff
@@ -8,7 +8,29 @@ placeholder
 
 ## Arian Ghorbani
 
-placeholder
+### Introduction
+
+By the end of the capstone, I would like to have worked on and completed the player identity server, which involves setting up the management of keys and VPN configurations used by players to connect to game environments. I’d like to be able to reference my work on Juno’s config management system on resumes after graduation.
+
+### System Architecture
```

Comment:
i dont want to include the figures for each individual section, because its the same information for each individual part. so i would take out the section headers here and include the included captions as a separate paragraph in the introduction section.